js/ui/tooltip-base.js

```diff
@@ -0,0 +1,171 @@
+import $ from "jquery";
+import Backbone from "backbone";
+Backbone.$ = $;
+
+import "../../css/ui/tooltips.css";
```

Comment:
I see, you're importing the .css directly in the .js files.  Does this result in the .css appearing in the resulting bundles?